gym_solo/core/test_obs_observations.py

```diff
@@ -1,20 +1,44 @@
 import unittest
 from gym_solo.core import obs
 
+from abc import ABC, abstractmethod
 from parameterized import parameterized
+from pybullet_utils import bullet_client
 from unittest import mock
 
-import numpy as np
 import math
+import numpy as np
+import pybullet as p
+
+
+class ObservationBaseTestCase(unittest.TestCase, ABC):
+  @property
+  @abstractmethod
+  def obs_cls(self):
+    """The observation class to instantiate."""
+    pass
+
+  def setUp(self):
+    self.client = bullet_client.BulletClient(connection_mode=p.DIRECT)
+
+  def tearDown(self):
+    self.client.disconnect()
+
+  def build_obs(self, *args, **kwargs):
+    o = self.obs_cls(*args, **kwargs)
```

Comment:
obs_cls does not allow for any arguments correct? Are you expecting subclasses will take care of it?

Reply:
since it's an abstract property, it's not actually a method. Rather, it's a `Callable` class. The arguments are the input arguments to the `Observation`. Like in the subclass:

https://github.com/WPI-MMR/gym_solo/blob/e7c17df8bf0a2011d9e9dff7efb5734fde142347/gym_solo/core/test_obs_observations.py#L33-L34

The arguments are the input arguments to `TorsoIMU`